src/thunderbird_accounts/mail/client.py

```diff
@@ -199,23 +222,23 @@ def create_account(self, email, username, uuid='', app_password=None):
         # Return the pkid
         return data.get('data')
 
-    def get_account(self, username):
-        response = self._get_principal(username)
+    def get_account(self, oidc_id: str):
```

Comment:
Previously the inbox name (unique lookup field) was the primary email address (keycloak's username field.) That's no longer true as we're looking up keycloak's uuid. 

To keep things generic in case we switch away from keycloak it's called oidc_id.